src/Tasks/AppConfig/AppConfig.cs

```diff
@@ -23,6 +24,11 @@ internal void Load(string appConfigFile)
             try
             {
                 var readerSettings = new XmlReaderSettings { DtdProcessing = DtdProcessing.Ignore };
+
+                // it's important to normalize the path as it may contain two slashes
+                // see https://github.com/microsoft/msbuild/issues/4335 for details.
+                appConfigFile = Path.GetFullPath(appConfigFile);
```

Comment:
```suggestion
                appConfigFile = FileUtilities.NormalizePath(appConfigFile);
```

? That's the one we generally use everywhere.